.github/Contributing.md

```diff
@@ -1,5 +1,10 @@
 ## Bash Style Conventions
 
+The most important thing re coding style (as with any contribution) is to
+follow the style conventions in place for the existing code.
+
+Here are some basics:
```

Comment:
How do you feel about using https://google.github.io/styleguide/shellguide.html as a basis? It does have different spacing conventions. Would actually generally be curious to see what you think about it.

Reply:
I've looked at it before. Just skimmed it again. Some of it I happen to agree with but most of it looks like it was just made up by some programmer who wasn't super experienced with Bash. Almost angers me enough to write a good one and to make   a show of why that "bash strict mode" is also bad.

Example. How is:
```
# Long commands
command1 \
  | command2 \
  | command3 \
  | command4
```
good compared to:
```
# Long commands
command1 |
  command2 |
  command3 |
  command4
  ```
  splitting lines is a good idea, but one of them comes from someone who learned Bash yesterday.